### PR TITLE
Feature/spa Àlex Barba

### PR DESCRIPTION
**Pull request by lexbarba**

### CODE REVIEW

File: staff/alex-barba/horroy/horroy.js [hunk 1]
```diff
@@ -0,0 +1,325 @@
...
+        if (this[i] === value){
+            return i;
+        }
+    }
+    return -1;
+};
+
+/* Abstraction of join */
+
+Horroy.prototype.join = function(separator) {
```
Comment (Arcoders) [inline staff/alex-barba/horroy/horroy.js]: missing arguments validation

Comment (lexbarba) [reply staff/alex-barba/horroy/horroy.js]: You were right... fixed and tested!

File: staff/alex-barba/spa/index.html [hunk 2]
```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
```
Comment (martimalek) [inline staff/alex-barba/spa/index.html:8]: You could put a title here, maybe something like "SPA".

Comment (lexbarba) [reply staff/alex-barba/spa/index.html:8]: I just updated ti to "Document", which is the title that I have thought for this great project. Thanks anyway!

**Pull request merged**